crates/primitives/src/signature/sig.rs

```diff
@@ -8,54 +8,34 @@ use crate::{
 use alloc::vec::Vec;
 use core::str::FromStr;
 
+/// The order of the secp256k1 curve
+const SECP256K1N_ORDER: U256 = U256::from_be_bytes(
+    fixed_bytes!("FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141").0,
+);
+
 /// An Ethereum ECDSA signature.
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
-pub struct Signature<T> {
-    /// Memoized ecdsa signature (if any)
-    inner: T,
-
```

Comment:
this makes sense, this wasn't very useful and would also result in a higher mem footprint

Reply:
https://github.com/alloy-rs/core/pull/719#discussion_r1741064689